src/shadowbox/server/server_access_key.ts

```diff
@@ -182,7 +188,9 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
       encryptionMethod,
       password,
     };
-    const accessKey = new ServerAccessKey(id, '', metricsId, proxyParams);
+    const name = params?.name ?? this.NEW_USER_DEFAULT_NAME;
+    const dataLimit = params?.dataLimit ?? this.NEW_USER_DEFAULT_DATA_LIMIT;
```

Comment:
You can remove this line, since the ServerAccessKey has the dataLimit as optional.

Also, the code reads as if we set a value other than `undefined`, which we don't.